ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/hibernate/jipijapa-hibernate/main/module.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.hibernate.jipijapa-hibernate">
```

Comment:
Note this was just a change to the module name from `org.hibernate.jipijapa-hibernate6`. The idea being in the future we just provision the module with which ever integration is needed and other modules don't need to change the dependency.

We could add an alias for the old module, but this is private and didn't seem like it would be needed.